### DISCUSSION TITLE
Linking to headers in different notebooks

### DISCUSSION THREAD

**Discussion by DKilkenny** [executablebooks/meta#337] [Q&A] [answered]
Hello,

I am looking for a way to link to a different notebook's header. 

For example I have notebook `A` and `B`. Inside notebook `A` I want to link to `# Header One` contained in notebook `B`. I can't seem to find a way to do that. Does this feature exist? I know that you could reference an internal header in `A` if you're in that notebook but that same syntax doesn't seem to carry to external headers.

**Comment by chrisjsewell** [comment, marked as answer]
Heya, so you can either create a "target" for that header, e.g. 

```
(my-reference)=
# Header One

[text](my-reference)
```

these reference are "project wide" (you will get a warning if any duplicates are found)

or use this "more advanced" sphinx configuration (I guess this is not documented in the jupyter-book documentation yet): https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#auto-generated-header-anchors

**Reply by DKilkenny** [reply]
Got it. Thank you!
